sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineOperationsTests.java

```diff
@@ -104,6 +108,38 @@ protected void cleanUpResources() {
         }
     }
 
+    @Test
+    @Disabled("The `userData` is not returned, so can not use `Assertions.assertEquals` be determine whether the returned `userData` is correctly.")
```

Comment:
Enabled the test anyway.

If userData is not returned, we should not provide the getter.

Some data is secret and not returned.

Reply:
Agree, getter is not needed.

Reply:
please check https://learn.microsoft.com/en-us/azure/virtual-machines/user-data

> User data can be queried via GET VM/VMSS API with $expand option.

seems userData can be get, but need some expand option?